runtimes/eden/src/constants.rs

```diff
@@ -73,7 +74,7 @@ pub const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 /// We allow for .5 seconds of compute with a 16 second average block time.
 pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
 	WEIGHT_REF_TIME_PER_SECOND.saturating_div(2),
-	polkadot_primitives::v4::MAX_POV_SIZE as u64,
+	polkadot_primitives::v5::MAX_POV_SIZE as u64,
```

Comment:
import MAX_POV_SIZE from a place which points ti the last version automatically

Reply:
done